autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>should it be "an s3 bucket" instead of "a s3 bucket"?</p><p> </p><p>...We need to have an s3 bucket to...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
done